services/submission.js

```diff
@@ -364,6 +364,30 @@ class Submission {
         }
         return this.dataRecordService.submissionQCResults(params._id, params.first, params.offset, params.orderBy, params.sortDirection);
     }
+    
+    async listBatchFiles(params, context) {
+        verifySession(context)
+            .verifyInitialized()
+            .verifyRole([
+                ROLES.ADMIN, ROLES.FEDERAL_LEAD, ROLES.CURATOR, // can see submission details for all submissions
+                ROLES.ORG_OWNER, // can see submission details for submissions associated with his/her own organization
+                ROLES.SUBMITTER, // can see submission details for his/her own submissions
+                ROLES.DC_POC // can see submission details for submissions associated with his/her Data Commons
+            ]);
+        const userRole = context.userInfo?.role;
+        let submission = null;
+        if ([ROLES.ORG_OWNER, ROLES.SUBMITTER, ROLES.DC_POC].includes(userRole)){
+            submission = (await this.submissionCollection.find(params.submissionID)).pop();
```

Comment:
no need to assign submission.

Reply:
Updated to use `verifyBatchPermission` function, but I believe we do need the submission to verify the permissions for a user to view a Batch for specific roles